### ISSUE TITLE
Gradle execution failed because no access token on version 1.1.0

### ISSUE THREAD

**Issue by readyiman** [mapbox/mapbox-maps-flutter#495]
Debug console:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve com.mapbox.maps:android:11.3.0.
     Required by:
         project :app > project :mapbox_maps_flutter
      > Could not resolve com.mapbox.maps:android:11.3.0.
         > Could not get resource 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/maps/android/11.3.0/android-11.3.0.pom'.
            > Could not GET 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/maps/android/11.3.0/android-11.3.0.pom'. Received status code 403 from server: Forbidden
```

link:
`{"message":"Not Authorized - No Token","url":"[GET] /downloads/v2/releases/maven/com/mapbox/maps/android/11.3.0/android-11.3.0.pom","auth":"<no access token>","ua":"Amazon CloudFront"}`

I had my public access token set on launch.json btw (if related), it works fine before.

**Comment by evil159** [comment]
Have you configured your secret token as described [here](https://github.com/mapbox/mapbox-maps-flutter?tab=readme-ov-file#secret-token)?

**Comment by readyiman** [comment]
I put my secret token at `~/android/gradle.properties` not at `~/.gradle/gradle.properties` as instructed because there is no `gradle.properties` file either at `~/.gradle` or `~/android/.gradle`. This way works fine before v1, I didn't change anything on it.

I did update Flutter to latest version that ask me to migrate Gradle as this instruction https://docs.flutter.dev/release/breaking-changes/flutter-gradle-plugin-apply

**Comment by Yorlandiscuba** [comment]
@readyiman You fix it?, i have the seem issue

**Comment by readyiman** [comment]
> @readyiman You fix it?, i have the seem issue

Nah. I don't know how to fix it. I'm only using mapbox just to try.

**Comment by Abdul-AhadKhan** [comment]
@readyiman Have you fixed the issue ?

**Comment by O-Hannonen** [comment]
We no longer require the `SDK_REGISTRY_TOKEN`. Closing this one as its most likely related to the secret token config. Feel free to re-open if there still are some issues!
